src/assets/style/components/listing.sass

```diff
@@ -1,3 +1,8 @@
+button
```

Comment:
There are rare occasions when it's necessary to redefine html elements like this but if you're doing it at a component level you're probably doing it wrong.  There's no reason not to use a class here.  However, bulma has a built in tab style.